modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -682,4 +683,26 @@ describe('browser.tinymce.core.UndoManagerTest', () => {
     });
   });
 
+  it('TINY-9222: Scroll to the cursor after undo and redo', () => {
+    const editor = hook.editor();
+    editor.undoManager.clear();
+    editor.focus();
+
+    const HEIGHT = 5000;
```

Comment:
This should be lower case as this uppercase naming format isn't used at Tiny for constants (there was a thread about this yesterday in a premium library PR). As @spocke there though a lint rule really should be added to enforce this so we probably need a tech debt Jira logged to follow up.

Reply:
Updated [here](https://github.com/tinymce/tinymce/pull/8205/commits/242f06e4430020e4e065f3ea6957594e18f5a9c1)